src/Http/Client/Request.php

```diff
@@ -92,7 +92,7 @@ protected function setContent(array|string $content)
         if (is_array($content)) {
             $formData = new FormData();
             $formData->addMany($content);
-            /** @phpstan-var array<non-empty-string, non-empty-string> $headers */
+            /** @psalm-var array<non-empty-string, non-empty-string> $headers */
```

Comment:
```suggestion
            /** @var array<non-empty-string, non-empty-string> $headers */
```

There are various other `non-empty-foo` uses in the diff which don't use the prefixed annotation.